app/controllers/category_transactions_controller.rb

```diff
@@ -0,0 +1,32 @@
+class CategoryTransactionsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_category, only: :new
+
+  def new
+  end
+
+  def create
+    @category_transaction = CategoryTransactions::Creator.new(category_transaction_params)
+    respond_to do |format|
+      if @category_transaction.call
```

Comment:
@marderer `#call` isn't required name, you can call it `#create` if it sounds better